### ISSUE TITLE
Sending special symbols and letters is possible

### ISSUE THREAD

**Issue by kerlikur** [buerokratt/Machine-Translation-to-Buerokratt#23]
AS AN End User and AS A CSA
I WANT TO send special symbols and letters, also see them in conversation
SO THAT I can communicate in correct language with symbols and letters specific to the language

### Acceptance Criteria:
- [ ] There must be possibility to send symbols and letters specific to the language.

**Comment by turnerrainer** [comment]
@kerlikur this requirement is a generic one and not related to machine translation. Obsolete as a separate issue but might be worth adding as an AC somewhere.

**Comment by kerlikur** [comment]
@turnerrainer 
This must be done before interfacing the machine translation.
